app/presenters/document/record_presenter.rb

```diff
@@ -0,0 +1,76 @@
+module Document
+  ##
+  # Blacklight document presenter for a Europeana record
+  class RecordPresenter < DocumentPresenter
+    def edm_resource_url
+      @edm_resource_url ||= @document.fetch('aggregations.edmIsShownBy', nil)
+    end
+
+    def media_web_resources(options)
```

Comment:
Assignment Branch Condition size for media_web_resources is too high. [30.41/15]<br>Method has too many lines. [14/10]